src/test/java/app/knock/api/model/WorkflowTriggerSerializeTests.java

```diff
@@ -56,13 +57,16 @@ void mappedPropertiesShouldUnwrap() throws JSONException {
     }
 
     @Test
-    void mappedPropertiesShouldUnwrapActorObject() throws JSONException {
+    void mappedPropertiesShouldUnwrapActorAndTenantObject() throws JSONException {
         WorkflowTriggerRequest workflowTrigger = WorkflowTriggerRequest.builder()
                 .key("new-feature")
                 .actor(WorkflowTriggerRequest.ObjectRecipientIdentifier.builder()
                         .id("object-id")
                         .collection("collection")
                         .build())
+                .tenant(TenantRecipientIdentifier.builder()
+                        .id("tenant-id")
```

Comment:
Seeing this in use, I don't think we need this builder as all we're ever doing is building a string?

Reply:
So there are three posible values for a tenant:
* A raw string referencing the tenant ID
* An object containing only an ID property(for this we use the `TenantRecipientIdentifier` builder we are talking about)
* An object with more properties

So from my understanding we need this builder for the second case, as I want to be clear a tenant can be an object with a simple ID and without more properties.

Reply:
your example of (2) is wholly covered under (3). We only had this builder for objects because you can explicitly build an object reference that needs an id and a collection, so the builder was a convenience method to build that elegantly, without needing to pass a map. 

For tenants, the interface should just be either a string, or a map.

Reply:
OK got it, I'll remove the builder.

Reply:
https://github.com/knocklabs/knock-java/pull/32/commits/40b72b87406d8b4a64ceacf546248ef4805b2c95